README.md

```diff
@@ -20,6 +20,14 @@
 
 **Release notes and unreleased changes can be found in the [CHANGELOG](CHANGELOG.md)**
 
+## Github Codespaces template
+
+This repo https://github.com/near/cargo-near-new-project-template contains project template
```

Comment:
```suggestion
This [repo](https://github.com/near/cargo-near-new-project-template) contains project template
```

But to be honest, I would make it something like that.

```md
* Quick start your project with the [template repository](https://github.com/near/cargo-near-new-project-template) generated by `cargo near new` command
* Try it out using [Github Codespaces devcontainers](Link to them). The containers allow you to start building without the need to install any dependencies on your machine and provide the actual version.
```

Reply:
the widget is some javascript illusion, there's no link to it